docs/conf.py

```diff
@@ -1,34 +1,25 @@
-# -- Project information -----------------------------------------------------
+# Project information
+project = "deshima-sensitivity"
+copyright = "2018-2020 DESHIMA software team"
+author = "Akira Endo"
+release = "0.2.4"
 
-project = 'deshima-sensitivity'
-copyright = '2020, Akio Taniguchi'
-author = 'Akio Taniguchi'
 
-
-# -- APIDOC location ---------------------------------------------------------
-import os
-import sys
-sys.path.insert(0, os.path.abspath('..'))
```

Comment:
Is this not needed to get the apidoc right?

Reply:
Thank you for reviewing the PR!

> Is this not needed to get the apidoc right?
As far as it is used from `etc/builddocs`, it is not necessary.

Reply:
OK! Good to know.